issue_drafts/issue_4.md

```diff
@@ -0,0 +1,126 @@
+# Issue 4: Generalizing Rules with Wildcards
+
+Pipeline steps: Compute the mean predicted temperature on each day of the year (at all depths) for each lake of interest
+
+Concepts learned:
+- wildcards
+- rule all
+- dry-run
+
+## Update `calc_doy_means.py` script to use properties from Snakemake rule
+Our current workflow is able to fetch and unzip raw lake temperature prediction data from ScienceBase. In the `out` file of our `1_fetch` step, we have a folder called `tmp` containing a set of csv files, each of which contains daily temperature predictions for a single lake (at multiple depths).
+
+The daily data spans nearly 40 years, and we would like to process it by calculating the day-of-year mean temperature predictions (at all depths) for each lake. Let's add a new rule to do this.
```

Comment:
We actually used to argue about this at my last job a lot. I was on team plural data, but our data journalist thought data should be treated as singular because that's how most non-technical people use it. I guess she influenced me because I subconsciously joined her team here 😆